cucumber/src/main/java/org/chodavarapu/datamill/cucumber/PropertySteps.java

```diff
@@ -54,7 +54,9 @@ public void captureUsingRegex(String regex, String existingProperty, String newP
 
     @Given("^a random alphanumeric (?:name|string) is stored as " + Phrases.PROPERTY_KEY + "$")
     public void generateRandomAlphanumeric(String key) {
-        propertyStore.put(key, RandomGenerator.generateRandomAlphanumeric(16));
+        if (!propertyStore.contains(key)) {
```

Comment:
Not sure why we want to do this? I think we just don't run this again if we don't want to change the random string stored with a certain property.